KafkaFlow.sln

```diff
@@ -0,0 +1,265 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.7.34031.279
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Framework", "Framework", "{068CB250-2804-4C7E-9490-17F432B9CE21}"
```

Comment:
I saw that projects in the solution were moved to folders, and this is a better organization.

But I think the main project shouldn't be inside a folder, otherwise, it can be harder to spot the main project. I would prefer something like this:
![image](https://github.com/Farfetch/kafkaflow/assets/3499967/84e79a08-0c0c-41c7-a165-023ee5918843)

Nevetheless, this is a matter of personal opinion, so I will leave the decision up to you on what you think its best.

Reply:
Just ignore this discussion, since this folder structure was already being used.